terraform/applications/moderator/iam.tf

```diff
@@ -0,0 +1,49 @@
+#
+# Moderator Developers ECR Read/Write Access Policy
+#
+data "aws_iam_policy_document" "ecr_rw" {
+  statement {
+    actions = [
+      "ecr:GetAuthorizationToken",
+    ]
+
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:GetRepositoryPolicy",
+      "ecr:DescribeRepositories",
```

Comment:
does this action work if we assign it to the ecr_arn?  I would expect it to be in the block with resources=*.

Reply:
i thought it just gave the description response for the one repo they have access to - moderator. i can't see a reason to give the devs the ability to describe all repositories. will test behavior shortly to confirm my understanding.

Reply:
@duallain you're right, describeRepositories only shows an error bc it has to run on all, not a filtered response type deal. will repair.

Reply:
(once this ns delete for moderator-stage completes)

Reply:
changed. thanks again for catching that.